dashboard/test/ui/features/platform/signing_in.feature

```diff
@@ -4,8 +4,8 @@ Feature: Signing in and signing out
 
 Scenario: Student sign in from code.org
   Given I create a student named "Bob"
-  And I set the cookie named "_loc_notice" to "1"
   And I sign out
+  And I set the cookie named "_loc_notice" to "1"
```

Comment:
I am being more strict about cookies after a sign-out such that setting a cookie like this is not possible beforehand. Some other tests will likely break if they rely on cookies to provide some testing state.